chaos-monkey-dependencies/pom.xml

```
@@ -26,7 +26,7 @@
     <modelVersion>4.0.0</modelVersion>
     <packaging>pom</packaging>
     <properties>
-        <h2.version>1.4.197</h2.version>
+        <h2.version>2.0.206</h2.version>
```

```suggestion
        <h2.version>${h2.version}</h2.version>
```
Instead of a fixed version, we should use the spring managed version. See https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#dependency-versions for more details